docs/styleguide.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         '../react/Input/index.jsx',
         '../react/Label/index.jsx',
         '../react/Radio/index.jsx',
-        '../react/SelectBox/index.jsx',
+        '../react/SelectBox/SelectBox.jsx',
```

Comment:
Do you test the component with the fixed option so?

Reply:
Since it is in the styleguide, it gets snapshotted.